src/TimeModeling/Modeling/utils.jl

```diff
@@ -1,10 +1,28 @@
-
-
-function judipmap(func, iter::UnitRange; )
+function judipmap(func, iter::UnitRange; on_error = nothing)
     # Ignore map and workers if single source
     length(iter) == 1 && (return [func(iter[1])])
-    # Switch to asyncmap if serial julia
-    length(default_worker_pool()) < 2 && (return asyncmap(func, iter))
     # Standard pmap if parallel and multiple sources
-    return pmap(func, iter)
+    return pmap(func, iter; distributed = (length(default_worker_pool()) >= 2), on_error=on_error)
 end
+
+function judipmap(func, model::Array{Model,1}, source::Array{judiVector{T,Array{T,2}},1}, dObs::Array{judiVector{T,Array{T,2}},1}, dm::Array) where T
```

Comment:
if `dm` is the same for all can be absorbed in `func` and remove this case.

Reply:
No `dm` here is an array of `dm`. Let me identify it as `dm::Union{Array{Array{T,1},1}, Array{PhysicalParameter{T},1}}`